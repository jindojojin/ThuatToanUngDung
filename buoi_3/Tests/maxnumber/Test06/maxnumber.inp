5555555555555555555555555555555555555551111111111111111111044444444444444444440111111111111111111111110000000000000000000000000000000000200000000000000000000000036000000000000000025000000000000000000000000000000000000000000000000000111111111111111111111111100222222222222222222222222222220
2222222222222222222222222222222222220000000000000000000000000000000000000000000000000000000000444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444
