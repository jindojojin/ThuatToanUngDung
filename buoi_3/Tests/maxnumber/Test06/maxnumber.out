444444444444444444422222222222222222222222222222220000000000000000000000000000000000000000000000000000000000
